modules/services/waybar.nix

```diff
@@ -0,0 +1,51 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.waybar;
+
+  configText = builtins.toJSON ({ inherit (cfg) order; } // cfg.extraConfig);
+
+  configFile = pkgs.writeText "config" configText;
+
+in {
+  meta.maintainers = [ maintainers.onny ];
+
+  options = {
```

Comment:
Are you certain that the module shouldn't have a config section? I assume that is a mistake…